hazelcast/generated-sources/src/hazelcast/client/protocol/codec/codecs.h

```diff
@@ -1733,231 +1489,190 @@ namespace hazelcast {
-                     *                  INVALIDATION(256)
-                     *                  LOADED(512)
+                     * @param old_value Old value of the entry event.
+                     * @param merging_value Incoming merging value of the entry event.
+                     * @param event_type Type of the entry event. Possible values are
+                     *                   ADDED(1)
+                     *                   REMOVED(2)
+                     *                   UPDATED(4)
+                     *                   EVICTED(8)
+                     *                   EXPIRED(16)
+                     *                   EVICT_ALL(32)
+                     *                   CLEAR_ALL(64)
+                     *                   MERGED(128)
+                     *                   INVALIDATION(256)
+                     *                   LOADED(512)
                      * @param uuid UUID of the member that dispatches the event.
-                     * @param numberOfAffectedEntries Number of entries affected by this event.
+                     * @param number_of_affected_entries Number of entries affected by this event.
                     */
-                    virtual void handle_entry(const boost::optional<serialization::pimpl::data> &key,
-                                              const boost::optional<serialization::pimpl::data> &value,
-                                              const boost::optional<serialization::pimpl::data> &old_value,
-                                              const boost::optional<serialization::pimpl::data> &merging_value,
-                                              int32_t event_type, boost::uuids::uuid uuid,
-                                              int32_t number_of_affected_entries) = 0;
+                    virtual void handle_entry( const boost::optional<serialization::pimpl::data> & key,  const boost::optional<serialization::pimpl::data> & value,  const boost::optional<serialization::pimpl::data> & old_value,  const boost::optional<serialization::pimpl::data> & merging_value, int32_t event_type, boost::uuids::uuid uuid, int32_t number_of_affected_entries) = 0;
 
                 };
 
                 /**
                  * Adds the specified entry listener for the specified key. The listener will be notified for all
                  * add/remove/update/evict events of the specified key only.
                  */
-                ClientMessage HAZELCAST_API replicatedmap_addentrylistenertokey_encode(const std::string &name,
-                                                                                       const serialization::pimpl::data &key,
-                                                                                       bool local_only);
+                ClientMessage HAZELCAST_API replicatedmap_addentrylistenertokey_encode(const std::string &name, const serialization::pimpl::data &key, bool local_only);
```

Comment:
should we also fix the method names here? `replicatedmap_addentrylistenertokey_encode` -> `replicated_map_add_entry_listener_to_key_encode`

Reply:
I agree with you, but I would need to change where these are used in the code and it's a bit more refactoring than I can do at the moment. Also I have better plans for the code generation that I'll try to later. So let's ignore this for now.